src/components/vote-form.js

```diff
@@ -1,10 +1,78 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components";
+import axios from "axios";
 
+import List from "./vote-list";
 export default function VoteForm() {
-  return <Wrapper>안녕 나는 투표 폼!</Wrapper>;
+  const [candidates, setCandidateList] = useState([]);
+  useEffect(() => {
+    getCandidateList();
+  }, []);
+  const getCandidateList = async () => {
+    await axios
+      .get(process.env.API_HOST + "/candidates/", candidates)
+      .then(({ data }) => {
+        setCandidateList(data);
+        console.log(candidates);
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
+  };
+  const Vote = (candidate) => {
+    axios
+      .put(
+        process.env.API_HOST + "/candidates/" + candidate.id + "/vote/",
+        voteCount
+      )
+      .then(({ data }) => {
+        alert(candidate.name + "님께 투표했습니다!");
+        getCandidates();
+      })
+      .catch(function (error) {
+        console.log(error);
+        alert("실패했습니다!");
+      });
+  };
+  let i = 1;
+  return (
+    <Wrapper>
+      <Title>
+        <RedTitle>프론트엔드 인기쟁이</RedTitle>는 누구?
+      </Title>
+      <Desc>CEOS 프론트엔드 개발자 인기순위 및 투표창입니다.</Desc>
+      <VoteSection>
+        {candidates
+          .sort((a, b) => {
+            return b.voteCount - a.voteCount;
+          })
+          .map((candidate) => (
+            <List
+              key={candidate._id}
+              id={candidate._id}
+              i={i++}
+              {...candidate}
+              getCandidateList={getCandidateList}
+            />
+          ))}
+      </VoteSection>
+    </Wrapper>
+  );
```

Comment:
```suggestion
  return (
    <Wrapper>
      <Title>
        <Red>프론트엔드 인기쟁이</Red>는 누구?
      </Title>
      <SubTitle>CEOS 프론트엔드 개발자 인기순위 및 투표창입니다.</SubTitle>
      <VoteSection>
        {candidates
          .sort((a, b) => {
            return b.voteCount - a.voteCount;
          })
          .map((candidate, index) => {
          const {_id : id} = candidate;
          return(
            <CandidateCard
              key={id}
              {...candidate}
              {...{getCandidateList}}
            />
          )})}
      </VoteSection>
    </Wrapper>
  );
```
1. 컴포넌트 이름들을 조금 더 명시적으로 바꿔봤어용
2. ```candidate``` 객체 안에 ```_id```를 ```id```라는 이름으로 destructuring 해서 쓰면 더 좋을 것 같아용 [참고](https://codeburst.io/renaming-destructured-variables-in-es6-807549754972)
3. ```index```는 ```map()```의 parameter에서 받아서 넘겨주면 됩니다.
4. ```id```가 ```candidate``` 안에 있어 spread operator로 넘겨주면 id가 넘어가서 따로 또 넘겨줄 필요는 없어요!
5. props 이름과 넘길 값을 담은 변수의 이름이 같다면 spread operator를 이용해 넘겨주면 좋아용
6. ```<List/>``` 컴포넌트 네이밍이 아쉬워요. 여러 명에 대한 정보가 담겨있지 않으니 List는 아닙니당. ```<CandidateCard/>```와 같이 네이밍 하는 건 어떨까용